pom.xml

```diff
@@ -36,9 +36,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <openmrsPlatformVersion>${project.version}</openmrsPlatformVersion>
-        <webservices.restVersion>2.17</webservices.restVersion>
-        <owaVersion>1.6.3</owaVersion>
-        <fhirVersion>1.7.0-SNAPSHOT</fhirVersion>
+        <webservices.restVersion>2.21.0</webservices.restVersion>
+        <owaVersion>1.8.1</owaVersion>
+        <fhirVersion>1.8.0-SNAPSHOT</fhirVersion>
```

Comment:
This is not the version that i mentioned on talk.

Reply:
ok, let me cross check.

Reply:
@dkayiwa I have tried to cross check but i dont seem to see anything different, please make me understand the version you want me to work on.

Reply:
You have 1.8.0-SNAPSHOT which is not what i mentioned on talk.

Reply:
oh ok let me work on that.